rules_daml/daml.bzl

```diff
@@ -368,6 +368,66 @@ def daml_compile_with_dalf(
         dalf = name + ".dalf",
     )
 
+def generate_and_track_dar_hash_file(name):
+    generate_dar_hash_file(name)
+
+    native.filegroup(
+        name = name + "-golden-hash",
+        srcs = native.glob([name + ".dar-hash"]),
+        visibility = ["//visibility:public"],
+    )
+
+    test_name = name + "-dar-hash-file-matches"
+
+    lbl = "//{package}:{target}".format(
+        package = native.package_name(),
+        target = test_name,
+    )
```

Comment:
this feels a bit silly but I couldn't find a better way to get the absolute label for the target being defined

Reply:
\* [Bazel 6.1](https://blog.bazel.build/2023/07/24/whats-new-with-bzlmod.html#changes-since-6-0) has [native.package_relative_label](https://bazel.build/rules/lib/toplevel/native#package_relative_label), but we're not there yet